src/System.Net.Http/src/System/Net/Http/Unix/CurlHandler.MultiAgent.cs

```diff
@@ -833,6 +839,39 @@ private static ulong CurlSendCallback(IntPtr buffer, ulong size, ulong nitems, I
                 return Interop.Http.CURL_READFUNC_ABORT;
             }
 
+            private static void SetLowSpeedLimit(EasyRequest easy, TimeSpan timeout)
+            {
+                // If there's no send/receive timeout on the handler, the defaults will
+                // remain disabled, and we don't need to worry about setting or resetting.
+                if (!easy._handler._hasSendReceiveTimeout)
+                {
+                    return;
+                }
+
+                // We need to ensure the timeout is set appropriately, which may include
+                // disabling a timeout currently set if one was.
+
+                long limit = 0, time = 0; // 0 == disable
+
+                if (timeout != Timeout.InfiniteTimeSpan)
+                {
+                    double secs = timeout.TotalSeconds;
+                    if (secs >= 1)
+                    {
+                        limit = 1;
```

Comment:
> I can't tell if you're arguing against the implementation or against the meaning of these timeouts

Me either. I'm not really trying to argue, just... am expression confusion.

Based on my sequencing bullets from earlier it seems like there is an off-by-one in the implementation (e.g. the ReceiveHeadersTimeout value is only applied after receiving the first header).  Otherwise I think I'm stuck in deciding if it feels like the implementation matches the spec, but I don't understand the spec.  So, failing there being a nice diagram explaining all the connection times and what timeouts are supposed to cover what parts, I'll just nod and smile at this point.